d2layouts/d2sequence/sequence_diagram.go

```diff
@@ -389,8 +389,9 @@ func (sd *sequenceDiagram) addLifelineEdges() {
 		sd.lifelines = append(sd.lifelines, &d2graph.Edge{
 			Attributes: &d2graph.Attributes{
 				Style: d2graph.Style{
-					StrokeDash:  &d2graph.Scalar{Value: fmt.Sprintf("%d", LIFELINE_STROKE_DASH)},
-					StrokeWidth: &d2graph.Scalar{Value: fmt.Sprintf("%d", LIFELINE_STROKE_WIDTH)},
+					StrokeDash:  actor.Attributes.Style.StrokeDash,  //&d2graph.Scalar{Value: fmt.Sprintf("%d", LIFELINE_STROKE_DASH)},
+					StrokeWidth: actor.Attributes.Style.StrokeWidth, //&d2graph.Scalar{Value: fmt.Sprintf("%d", LIFELINE_STROKE_WIDTH)},
```

Comment:
I think we need to check if each of these was manually set or not. the default values should be used here because nothing was set on the actors:

![Screen Shot 2023-04-06 at 2 06 46 PM](https://user-images.githubusercontent.com/85081687/230494098-0417688b-cc25-49b9-af34-a73aee96d2f9.png)

Reply:
@gavin-ts - Yep, I saw the diffs and realised that 👍 Will update!

Reply:
@gavin-ts - updated now! Diff looks much better now 👍